homeassistant/components/tellduslive/entry.py

```diff
@@ -119,7 +119,7 @@ def device_info(self):
         return {
             'identifiers': {('tellduslive', self.device.device_id)},
             'name': self.device.name,
-            'model': device['model'].title(),
-            'manufacturer': device['protocol'].title(),
+            'model': device.get('model', '').title(),
```

Comment:
Sorry, this won't solve it, since the key isn't missing in the other case. It returns `None`. So we have to check for `None` before applying the title op.

Reply:
Thanks for spotting this! I'll update the code.

Reply:
What is the preferred way for HA when the property is not available, should I set it to `None` `''` or not have it set at all?

```suggestion
            'model': device.get('model').title() if device.get('model') else None,
```